certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/config/CertificateDataConfigTypes.java

```diff
@@ -0,0 +1,8 @@
+package se.inera.intyg.certificateservice.application.certificate.dto.config;
+
+public enum CertificateDataConfigTypes {
```

Comment:
What you think of us removing the types that we don't support?

Reply:
I agree!